lib/routes/account.js

```diff
@@ -1419,9 +1419,10 @@ module.exports = function (
                       uid: uidHex,
                       code: request.payload.code
                     })
-                    return log.activityEvent('account.confirmed', fakeRequestObject, {
+                    log.activityEvent('account.confirmed', fakeRequestObject, {
```

Comment:
Not only is waiting for the log to return unnecessary here, returning the promise is actually wrong. If the activity event fails for some reason, it would cause us to log an `account.signin.confirm.invalid` error.